src/loc/CertificateAndLimits.tsx

```diff
@@ -45,6 +49,11 @@ export default function CertificateAndLimits(props: Props) {
         }
     }, [ api, props.loc ]);
 
+    const hasVoteFeature = useMemo(() => {
+        const legalOfficer = props.loc.ownerAddress;
+        return legalOfficer !== undefined && backendConfig[legalOfficer] && backendConfig[legalOfficer].features && backendConfig[legalOfficer].features.vote;
```

Comment:
I think turning `CommonContext.backendConfig` into a function would allow to factor and avoid repeating this boiler-plate code:
`backendConfig: ((legalOfficerAddress: string | undefined) => BackendConfig)`

Reply:
Agreed.